src/components/comment/Comment.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import Avatar from '../avatar';
+import Media from '../media/Media';
+import ActionDropdown from './ActionDropdown';
+
+import './Comment.scss';
+
+export enum CommentType {
+    COMMENT = 'comment',
+    ANNOTATION = 'annotation',
+}
+
+export interface CommentMention {
+    userName: string;
```

Comment:
Can we alphabetize the attributes of our types and props?

Reply:
Done